benches/bench.rs

```diff
@@ -73,6 +73,7 @@ fn bench_division<M: Measurement>(group: &mut BenchmarkGroup<'_, M>) {
     });
 }
 
+#[allow(deprecated)]
```

Comment:
Why not change it to use the non-deprecated method?

Reply:
Certainly could. The intent was to avoid touching existing code unnecessarily, but if it's better to take care of it all now, will do.